server/datastore/mysql/software_titles_test.go

```diff
@@ -570,7 +571,20 @@ func testTeamFilterSoftwareTitles(t *testing.T, ds *Datastore) {
 	require.NoError(t, err)
 	// ListSoftwareTitles does not populate version host counts, so we do that manually
 	titles[0].Versions[0].HostsCount = ptr.Uint(1)
-	assert.Equal(t, titles[0], fleet.SoftwareTitleListResult{ID: title.ID, Name: title.Name, Source: title.Source, Browser: title.Browser, HostsCount: title.HostsCount, VersionsCount: title.VersionsCount, Versions: title.Versions, CountsUpdatedAt: title.CountsUpdatedAt})
+	assert.Equal(
+		t,
+		titles[0],
+		fleet.SoftwareTitleListResult{
+			ID:              title.ID,
+			Name:            title.Name,
+			Source:          title.Source,
+			Browser:         title.Browser,
+			HostsCount:      title.HostsCount,
+			VersionsCount:   title.VersionsCount,
+			Versions:        title.Versions,
+			CountsUpdatedAt: title.CountsUpdatedAt,
+		},
+	)
```

Comment:
note: integration tests already cover both endpoints automatically, that's why they weren't changed.